otj-kafka/src/main/java/com/opentable/kafka/logging/OTKafkaHeaders.java

```diff
@@ -13,46 +13,52 @@
  */
 package com.opentable.kafka.logging;
 
-import java.util.Optional;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
 
 import com.opentable.conservedheaders.ConservedHeader;
 
 /**
- * Represents the names of various headers in Kafka. Ones marked appropriately map to a conserved header
+ * Represents the names of various headers in Kafka.
+ * These are namespaced to avoid collisions, except for request Id
  */
 public enum OTKafkaHeaders {
+
     REFERRING_HOST("ot-kafkalibrary-referring-host"),
```

Comment:
These are some quite verbose header names.

Reply:
Yeah they are. I wanted to namespace them but maybe I went a little crazy. Counterproposal?
Maybe just ot-kafka- (cut the library part)?

Reply:
@SegFaultAX